language-support/ts/daml-types/index.ts

```diff
@@ -30,18 +43,43 @@ export interface Template<T extends object, K = unknown, I extends string = stri
 }
 
 /**
- * Interface for objects representing DAML choices. It is similar to the
- * `Choice` type class in DAML.
+ * Interface for objects representing DAML choices.
+ *
+ * It is similar to the `Choice` type class in DAML.
```

Comment:
There is actually no `Choice` typeclass anymore (only a constraint synonym). And even if there is, we never advertised this in DAML and treated it more like an implementation detail so I would just remove this.

Reply:
ok, then let's remove this line.